README.md

```diff
@@ -21,7 +23,7 @@ flutter pub add webp --dev
 
 Add to your pubspec.yaml
```

Comment:
Add a period to the end of the sentence. And maybe make the filename an inline code block